### DISCUSSION TITLE
Acquistare saggiamente, in vista di futuri upgrade

### DISCUSSION THREAD

**Discussion by Buff81** [indomus/forum#642] [Hardware & Firmware]
Ciao a tutti, ho bisogno di una mano e di un consiglio. :-) 
Quest'anno ho comprato e ristrutturato un appartamento, avevo già un semplice echo dot che usavo per funzioni base (il massimo che c'avevo fatto era comandare una presa smart per accendere e spegnere ad orari fissi l'albero di natale) ma adesso che la casa è mia ho in programma di domotizzare (pian piano) il possibile (ho anche una tenda elettrica della Somfy, ma temo dovrò farmi un secondo telecomando e modificarlo con contatti puliti, si vedrà). 
Ho già montato due relè Shelly 1 su due luci ed ho collegato al motore dell'unica tapparella elettrificata il modulo shelly. Pian piano (quando monterò lampadari e tutte le luci) monterò i relè ovunque: mi arrangio con le mani e non è un problema tirare un cavo in più nel corrugato per portare una fase dove serve, per nascondere il relè nella plafoniera o nella base del lampadario (l'impianto è stato rifatto dove serviva, un po' di scatole sono ancora quelle originali e sono un po' ingolfate, non riesco a metterci direttamene il modulo).
Ho capito che il modo migliore per gestire le componenti sarà comprare un Raspberry e farci girare Home Assistant, ma credo che ci penserò dopo che i relè saranno montati: per ora mi arrangio con Alexa. Vorrei provare a montare un sensore di movimento che accenda la luce del bagno nelle ore notturne (e che poi la spenga dopo X secondi di inattività).  Mi sembra di capire che la scelta più sensata sarebbe un sensore di movimento che usi il protocollo Zigbee, ma di cui per ora non ho niente. Cosa consigliate per poterlo gestire attualmente con Alexa ma nell'ottica della futura implementazione di HA?

grazie per qualunque consiglio!

**Comment by KC-inDomus** [comment]
saggiamente, parti prima da Home Assistant. period. ;)
https://indomus.it/focus/domotica-si-parte-dalle-fondamenta-non-dal-tetto/

**Reply by Buff81** [reply]
"Temevo" una risposta simile :-) Chiedevo proprio perchè lo scopo finale a cui tendo è usare HA, ma se in "qualche modo" riesco ad ottenere qualche un risultato prima... tanto meglio. Se compro qualcosa, devo essere sicuro che funzionerà bene con HA. Il problema attuale è che i fine settimana sono ancora dedicati alla manovalanza fisica di casa (e non all'implementazione software): il prossimo step sarà montare tende e gli zoccoli in cucina, poi lampadari e luci. Nel momento in cui monto lampadari/faretti/dimmer e compagnia monterò direttamente i relè wireless, per fare il lavoro una volta. Di questo passo troverò il tempo per HA verso novembre... :-(
Se la soluzione non c'è, aspetterò e continuerò a non centrare l'interruttore del bagno di notte: di questo passo dovrò ritoccare il muro intorno, si sta già sporcando dalle manate 🤣
